docs/en/fuse-sdk/Overview.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: global
+title: Overview
+group: FUSE SDK
+priority: 1
+---
+
+The Alluxio POSIX API is a feature that allows mounting training datasets
+in specific storage services (e.g. S3, HDFS) to the local filesystem
+and provides caching capabilities to speed up I/O access to frequently used data.
+
+* Table of Contents
+  {:toc}
+
+## Local Cache vs Distributed Cache
+
+There are two kinds of caching capabilities: 1. local caching only 2. local caching + distributed caching.
+
+Differences between the two solutions are listed below, choose your desired solution based on training requirements and available resources.
+<table class="table table-striped">
```

Comment:
you know that markdown supports table format so you don't need to create a html table? https://www.markdownguide.org/extended-syntax/#tables

Reply:
too lazy to change lol this is added in previous PR